### PR TITLE
Addition of MrBayes

### PR DESCRIPTION
**Pull request by markiskander**
I added MrBayes tool/package along with working wrapper and testing. Let me know if anything needs altering. The one thing I can think of that's probably wrong is the citation that I added. Couldn't figure out how to extract a DOI from it (Pubmed's own tool failed to detect the article for some reason)

### CODE REVIEW

File: tools/mrbayes/mrbayes.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0"?>
+<tool id="mrbayes" name="MrBayes" version="3.2.6">
+  <description>with options and commands</description>
+  <requirements>
+    <requirement version="3.2.1" type="binary">mb</requirement>
```
Comment (markiskander) [inline tools/mrbayes/mrbayes.xml]: Also not sure if this is currently required or not. The tool_dependencies.xml installs mb, but I'm not sure if this is properly checking for it or not

File: tools/mrbayes/mrbayes.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+#silent sys.stderr.write("\n \n \n ======== Cheetah template variables ========\n")
+#for k,v in $searchList[2].items()
+#silent   sys.stderr.write(" %s = %s\n" % (str(k), str(v) ))
+#end for
+#silent sys.stderr.write("======== End of Cheetah template variables ========\n \n \n")
+  </command>
+  <configfiles>
+    <configfile name="command_file">
+        begin mrbayes;
+        set seed=$seed;
```
Comment (markiskander) [inline tools/mrbayes/mrbayes.xml]: I'll mark the things that I changed. This is one of them

File: tools/mrbayes/mrbayes.xml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    <param name="plot" type="boolean" truevalue="plot" falsevalue="" label="Include plot?"
+        help="Creates an x-y graph of the parameter over the course of the chain."/>
+
+    <param name="checkfreq" type="integer" label="&lt;hr&gt;Checkpoint frequency" value="100000" >
+       <validator type="in_range" min="100" max="inf" message="Must be greater than or equal to 100"/>
+       <help>Frequency with which checkpoints are written. Used for continuing analysis with append.</help>
+    </param>
+    <param name="append" type="boolean" truevalue="mcmcp append=yes" falsevalue="mcmcp append=no" label="Append to last checkpoint?"
+        help="Requires .ckp file (i.e. that number of generations > checkpoint frequency for previous analysis)." />
+    <param name="seed" type="integer" label="&lt;hr&gt;Seed" optional="false" value="5"
```
Comment (markiskander) [inline tools/mrbayes/mrbayes.xml]: Also added seed as a parameter

File: tools/mrbayes/mrbayes.xml (hunk 1 above)
Comment (ericenns) [inline tools/mrbayes/mrbayes.xml]: Maybe it isn't working since the dependency is 3.2.6 not 3.2.1

**Pull request merged**